packages/sdk-core/examples/custom-lexicon/README.md

```
@@ -0,0 +1,75 @@
+pnpm install
+
+# Run the example
+pnpm tsx examples/custom-lexicon/usage.ts
+
+# Run tests
+pnpm test examples/custom-lexicon
+```
+
+## Example Output
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)